app/services/shopify_import/data_savers/products/product_base.rb

```diff
@@ -23,19 +23,27 @@ def add_option_types
 
         def create_option_types
           option_types.map do |option_type, option_values|
-            spree_option_type =
-              Spree::OptionType.where('lower(name) = ?', option_type).first_or_create!(name: option_type,
-                                                                                       presentation: option_type)
+            spree_option_type = find_or_create_option_type(option_type)
+
             create_option_values(spree_option_type, option_values)
             spree_option_type.id
           end
         end
 
+        def find_or_create_option_type(option_type)
+          option_type_name = option_type.downcase
```

Comment:
You are storing the values in the db without making them lowercase, but then we only query for lowercase values here. This will not work.

Reply:
@vfonic I am string them as `lowercase`. Look at line 36 ```name: option_type_name```